packages/plugins/rubygem-smart_proxy_salt/rubygem-smart_proxy_salt.spec

```diff
@@ -23,19 +23,23 @@
 %global gem_name smart_proxy_salt
 %global plugin_name salt
 
-%global foreman_proxy_min_version 1.25
+%global foreman_proxy_min_version 2.5.2
 %global foreman_proxy_dir %{_root_datadir}/foreman-proxy
 %global foreman_proxy_statedir %{_root_localstatedir}/lib/foreman-proxy
 %global foreman_proxy_bundlerd_dir %{foreman_proxy_dir}/bundler.d
 %global foreman_proxy_settingsd_dir %{_root_sysconfdir}/foreman-proxy/settings.d
 %global smart_proxy_dynflow_bundlerd_dir %{_datadir}/smart_proxy_dynflow_core/bundler.d
 
 %global salt_config_dir %{_root_sysconfdir}/salt
+%global salt_config_runners_dir %{foreman_proxy_statedir}/salt/runners
+%global salt_config_reactors_dir %{foreman_proxy_statedir}/salt/reactors
+%global salt_config_grains_dir %{foreman_proxy_statedir}/salt/grains
+%global salt_minion_auth_install_dir %{buildroot}%{gem_instdir}/salt/minion_auth
```

Comment:
You only use this in the build step. I'm not sure a separate macro makes that much sense.

Reply:
File permissions are set [here](https://github.com/theforeman/foreman-packaging/pull/6968/files#diff-7f2b45922f25a10c95613ac59ae09ca985b02817e57bc926206b8815a454747aR194) as well, so its used at least two times. IMHO macros give a better overview of the paths used and reduce redundancy. Is there any downside of using them?

Reply:
I was just talking about this single line. For the others (salt_config_*) I agree they do make sense.

Reply:
Implemented as suggested.